src/Tests/Integration/fixtures/banks/isracard/02-pre-login/frame-2.html

```
@@ -5,7 +5,7 @@
   </head>
   <body style="background-color: transparent">
     <img
-      src="https://adservice.google.com/ddm/fls/z/src=16046083;type=rmrkt;cat=amexi0;rcb=18;ord=3357080771355;npa=0;auiddc=*;pscdl=noapi;frm=0;user_data_mode=a;_tu=KFA;gtm=45fe6640h2v9234171982z872195715za20gzb6529056zd6529056xea;gcd=13l3l3l3l1l1;dma=0;dc_fmt=2;tag_exp=0~115616985~115938466~115938468~118131809~118228214~118395334~119027224;epver=2;em=tv.*;dc_random=1780656042_XwlLrW1jktp6PlcJqVilVdCdi70oPmInGw;_dc_test=1;~oref=https%3A%2F%2Fdigital.isracard.co.il%2Fpersonalarea%2FLogin%2F"
+      src="https://adservice.google.com/ddm/fls/z/src=16046083;type=rmrkt;cat=amexi0;rcb=18;ord=3357080771355;npa=0;auiddc=*;pscdl=noapi;frm=0;user_data_mode=a;_tu=KFA;gtm=45fe6640h2v9234171982z872195715za20gzb6529056zd6529056xea;gcd=13l3l3l3l1l1;dma=0;dc_fmt=2;tag_exp=0~[redacted-id]~[redacted-id]~[redacted-id]~[redacted-id]~[redacted-id]~[redacted-id]~[redacted-id];epver=2;em=tv.*;dc_random=1780656042_XwlLrW1jktp6PlcJqVilVdCdi70oPmInGw;_dc_test=1;~oref=https%3A%2F%2Fdigital.isracard.co.il%2Fpersonalarea%2FLogin%2F"
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)